ballerina/listener.bal

```diff
@@ -74,6 +94,16 @@ public class Listener {
                 return error Error("Error occurred while detaching the service", result);
             }
         }
+        websocket:Listener? wsListener = self.wsListener;
+        if wsListener is websocket:Listener {
+            UpgradeService? wsService = getWebsocketServiceFromGraphqlService(s);
```

Comment:
Why do we need an interop function to get the websocket service? It is a class variable, right?

Reply:
This is to get the `UpgradeService`. It's not a class variable.